system/README.md

```diff
@@ -0,0 +1,182 @@
+# System Monitor for Autoware
```

Comment:
Please remove `system/README.md`. The file you should modify is  `system/system_monitor/README.md`.

Reply:
done